<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="btn_gray">
           <property name="text">
            <string>灰度化</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_binary">
           <property name="text">
            <string>二值化</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_gray_rev">
           <property name="text">
            <string>灰度反转</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_gamma">
           <property name="text">
            <string>伽马变换</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_bit_levels">
           <property name="text">
            <string>比特平面分层</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_histogram_equalize">
           <property name="text">
            <string>直方图均衡</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_hist_mattch">
           <property name="text">
            <string>直方图匹配</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>傅里叶变换 </string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>576</width>
           <height>713</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="Canvas" name="label_img1">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>请先加载图像</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea_2">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>576</width>
           <height>713</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="Canvas" name="label_img2">
            <property name="text">
             <string>处理后图像</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="action_cursor"/>
    <addaction name="actiondraw_line"/>
    <addaction name="actiondraw_rect"/>
    <addaction name="actiondraw_cicle"/>
    <addaction name="actioncolor"/>
    <addaction name="separator"/>
    <addaction name="actioncolor_2"/>
    <addaction name="actioncuxi"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_open">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="action_cursor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>光标</string>
   </property>
  </action>
  <action name="actiondraw_line">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>画线</string>
   </property>
  </action>
  <action name="actiondraw_rect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>画矩形</string>
   </property>
  </action>
  <action name="actiondraw_cicle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>画圆</string>
   </property>
  </action>
  <action name="actioncolor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>橡皮擦</string>
   </property>
  </action>
  <action name="actioncolor_2">
   <property name="text">
    <string>颜色选择</string>
   </property>
  </action>
  <action name="actioncuxi">
   <property name="text">
    <string>粗细</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Canvas</class>
   <extends>QLabel</extends>
   <header location="global">canvas.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
